packages/ckeditor5-coremedia-blocklist/src/lang/blocklist.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Localizations for Blocklist Plugin.
+ *
+ * @packageDocumentation
+ */
+import { add } from "@ckeditor/ckeditor5-utils/src/translation-service";
+
+add("de", {
```

Comment:
The wording in german feels inconsistent:

Block => Sperren (why not blockieren?)
zur Blockliste hinzufügen => Blockliste , here it is a blockliste and which is closer to blockieren.
Begriff zur Liste hinzufügen => Now it is only a "Liste". Could be "Wort blockieren".

Reply:
The wording is a Design department specification.